### PR TITLE
rabbitmq: Override PATH so rabbitmqctl works as expected

### PR DESCRIPTION
**Pull request by jeckersb**

### CODE REVIEW

**Review by Akrog — APPROVED** [review]
Thanks for the fix!
I think it is unlikely for the rabbitmq container to suddenly change its PATH and lead to issues with this override.

File: pkg/openstack/rabbitmq.go [hunk 1]
```diff
@@ -136,6 +136,21 @@ func reconcileRabbitMQ(
...
+									// the user running the command is either root or rabbitmq.  Inside
+									// of an openshift pod, however, the user is neither of these, so
+									// the wrapper script will always fail.
+
+									// By putting the real programs ahead of the wrapper in PATH we can
+									// avoid the unnecessary check and just run things directly as
+									// whatever user the pod has graciously generated for us.
+									Name:  "PATH",
+									Value: "/usr/lib/rabbitmq/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+								},
```
Comment (SeanMooney) [inline pkg/openstack/rabbitmq.go:153]: the alternitive would be to use tcib_envs: like this
https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/base.yaml#L43C1-L45
to set this in 
https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/rabbitmq/rabbitmq.yaml

**Comment by jeckersb** [comment]
recheck

**Review by abays — APPROVED** [review]
/lgtm

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
`openstack-operator-crc-podified-edpm-baremetal` and `openstack-operator-crc-podified-edpm-deployment` jobs have succeeded independently across separate test attempts.  Overriding Zuul.

**Review by dprince — APPROVED** [review]

**Comment by abays** [comment]
Debug pods used for CRC storage cleanup keep hiccuping during testing.

/test openstack-operator-build-deploy-kuttl

**Comment by abays** [comment]
```
Unable to connect to the server: read tcp 10.131.113.228:51860->38.102.83.117:6443: read: connection reset by peer
```

/test openstack-operator-build-deploy-kuttl

**Comment by abays** [comment]
> @jeckersb: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/openstack-operator-build-deploy-kuttl 	[c63d1e3](https://github.com/openstack-k8s-operators/openstack-operator/commit/c63d1e37cf4de1ebcaa61e4c1ab3ebb9dcaf2988) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_openstack-operator/415/pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl/1686820139053355008) 	true 	`/test openstack-operator-build-deploy-kuttl`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openstack-k8s-operators&repo=openstack-operator&pr=415). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:jeckersb).

This time the bundle failed to build, which has succeeded on all previous attempts...

```
+ echo 'Build openstack-operator-bundle failed to complete. Current status is Running. Aborting...'
Build openstack-operator-bundle failed to complete. Current status is Running. Aborting...
```

We just can't win here :)

/test openstack-operator-build-deploy-kuttl

**Pull request merged**